### ISSUE TITLE
How to get the model's test-dev result? Is that right if I change or edit some coding lines in test_net.py?

### ISSUE THREAD

**Issue by Alan-D-Chen** [zylo117/Yet-Another-EfficientDet-Pytorch#576]
The more detailed your answer, the better. Thanks a lot. (I am a new student for CV and _DL）Any help is appreciated!

**Comment by Alan-D-Chen** [comment]
LOOK at Table 2: EfficientDet performance on COCO [24] – Results are for single-model single-scale. test-dev is the COCO
test set and val is the validation set. Params and FLOPs denote the number of parameters and multiply-adds

**Comment by zylo117** [comment]
test set is almost the same as val set. just run the evaluation with coco_eval.py and upload the result json file to coco website

**Comment by Alan-D-Chen** [comment]
OK，I am going to trying~
